code/drasil-code/lib/Language/Drasil/Code/Imperative/WriteReadMe.hs

```diff
@@ -33,3 +36,9 @@ makeReadMe ReadMeInfo {
 makeInstr :: ImplementationType -> [FilePath]-> Doc
 makeInstr Library _ = empty
 makeInstr Program cfp = instDoc cfp
+
+-- |Helper that checks if the field is empty; hels with allowing optional content 
+-- rendering in a README file
+fieldEmptySTR :: String -> Maybe String
```

Comment:
Surely such a function already exists?

Reply:
updated in 24b0ddd47c50a8c21e91729ab7462e6da01d7e37